### DISCUSSION TITLE
Wrong contribution

### DISCUSSION THREAD

**Discussion by suikei-wang** [community/community#21649] [New to GitHub]
<p>Hi there,</p>
<p>I just found that my github profile shows wrong contribution after I changed my main email address.<br>
<em>I make sure I turned on the contribution for private repo</em><br>
I keep pushing to one private repo this year, and after I changed my main email address, I found it doesn’t show the contribution in my profile anymore. Also, my contributions to my homepage (a public repo) have also gone! And I just found that the update in my repo becomes:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/b/cbdb7dfae5ab9f166ade4d61cdd5d70676dd1cff.png" data-download-href="/uploads/short-url/t5pbxFfs7soDf6L8as3nL8rhIET.png?dl=1" title="Screen Shot 2021-04-26 at 14.08.55" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/19282010/181065998-1ca1228b-58bb-4960-91b8-8ec0941baae3.png" alt="Screen Shot 2021-04-26 at 14.08.55" data-base62-sha1="t5pbxFfs7soDf6L8as3nL8rhIET" width="690" height="36" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cbdb7dfae5ab9f166ade4d61cdd5d70676dd1cff_2_690x36.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cbdb7dfae5ab9f166ade4d61cdd5d70676dd1cff_2_1035x54.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cbdb7dfae5ab9f166ade4d61cdd5d70676dd1cff_2_1380x72.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cbdb7dfae5ab9f166ade4d61cdd5d70676dd1cff_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-04-26 at 14.08.55</span><span class="informations">1808×96 16.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
I don’t know why it shows my name twice and I lose all these contribution :((
<p>Thank you folks!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That probably means you didn’t change your author email address on the computer where you made the commits. GitHub uses that to assign commits to an account, so if you remove an email address from your account contributions using the removed address won’t be recognized.</p>
<p>You can check the author email configured on your computer with this command:</p>
<pre><code class="lang-auto">git config user.email
</code></pre>
<p>As for the double name, did you do merge a PR using the “rebase and merge” method?</p>

**Comment by suikei-wang** [comment]
<p>Fantastic! I add back my previous email address and it works. But my previous education email may be expired after a few weeks but I think it’s okay to leave it there.<br>
The double name is also solved after that. It helps a loooot! THANK YOU!!</p>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="suikei-wang" data-post="3" data-topic="176685">
<div class="title">
<div class="quote-controls"></div>
 suikei-wang:</div>
<blockquote>
<p>But my previous education email may be expired after a few weeks but I think it’s okay to leave it there.</p>
</blockquote>
</aside>
<p>For previous commits it should be, but I’d recommend changing the address you use for new commits. :wink:</p>
